ambry-router/src/test/java/com.github.ambry.router/NonBlockingRouterTest.java

```diff
@@ -74,9 +73,19 @@ private void setOperationParams() {
   @Test
   public void testNonBlockingRouterFactory()
       throws Exception {
-    Properties props = getNonBlockingRouterProperties();
-    VerifiableProperties verifiableProperties = new VerifiableProperties((props));
     MockClusterMap mockClusterMap = new MockClusterMap();
+    try {
+      Properties props = getNonBlockingRouterProperties("NotInClusterMap");
```

Comment:
minor. You can take out the creation of props and verifiable props outside of try block and over-ride it for next test case